x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -160,11 +165,19 @@ export class ManifestManager {
     schemaVersion: string;
   }): Promise<WrappedTranslatedExceptionList> {
     if (!this.cachedExceptionsListsByOs.has(`${listId}-${os}`)) {
-      const itemsByListId = await getAllItemsFromEndpointExceptionList({
-        elClient,
-        os,
-        listId,
-      });
+      let itemsByListId: ExceptionListItemSchema[] = [];
+      if (
+        (listId === ENDPOINT_HOST_ISOLATION_EXCEPTIONS_LIST_ID &&
```

Comment:
This is ok, but I don't think its needed. 

`endpointResponseActions`, which is part of "Endpoint Complete" is a product tier that inherits from "Endpoint Essentials", so `endpointArtifactManagement` is already enabled at that point. so just checking `endpointArtifactManagement` should be enough

Reply:
Hey, the check on the `appFeatures` does not take the user role into account, so users with no endpoint-related subFeature authorization will pass this check. I am not sure if that matters here, just to let you know.

Reply:
Thanks @semd . Yes, it does not matter here. This is a process that runs on the server side as a background task - so not under a specific user.